packages/blade/src/components/Input/TextInput/TextInput.stories.tsx

```diff
@@ -256,6 +256,23 @@ export const TextInput = TextInputTemplate.bind({});
 // Need to do this because of storybook's weird naming convention, More details here: https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#single-story-hoisting
 TextInput.storyName = 'TextInput';
 
+export const TextInputTypeNumber = TextInputTemplate.bind({});
+TextInputTypeNumber.storyName = 'TextInput with type number';
+TextInputTypeNumber.args = {
+  type: 'number',
+  label: 'Enter Number',
+  placeholder: 'Enter any random number',
+};
+
+// @ts-expect-error: Just another undocumented, untyped storybook property ;__;
+TextInputTypeNumber.story = {
```

Comment:
Dude how do you even find such properties? What is this sorcery?

Also, hope its not crashing on react native as some unsupported property or something 🙈

Reply:
> Dude how do you even find such properties? What is this sorcery?

Lol github issue comments ftw https://github.com/storybookjs/storybook/issues/8527#issuecomment-584946787

> Also, hope its not crashing on react native as some unsupported property or something 🙈

My XCode asking me to reinstall and that is failing lol so can't check. Hopefully it won't since `Component.anything = {}` is fair assignment 🙈

Reply:
ok finally xcode worked. checked the story its working.